File: iOS/PalindromeApp/Random/Random.swift
//
//  Random.swift
//  Random
//
//  Created by Aferdita Muriqi on 11/17/17.
//  Copyright © 2017 DITA. All rights reserved.
//

import Foundation

extension Array {
  public var randomObject: Element!{
    if (self.count > 0) {
      let random = Int(arc4random_uniform(UInt32(self.count)))
      return self[random]
    }
    else {
      return nil
    }
  }
}

File: Android/PalindromeApp/random/src/main/java/com/aferdita/random/Random.kt
package com.aferdita.random

import java.util.*

/**
 * Created by aferditamuriqi on 11/17/17.
 */


public val Array<*>.randomObject: Any?
    get() {
        if ((this.size > 0)) {
            val random = Random().nextInt(this.size)
            return this[random]
        } else {
            return null
        }
    }


